### PR TITLE
fix(forms): disable object links during quarterly closure

### PR DESCRIPTION
**Pull request by luytena**

### CODE REVIEW

**Comment by velrest** [comment]
@luytena Feel assigned for review

File: addon/components/quarterly-closure/disabled-form-info.hbs [hunk 1]
```diff
@@ -1,4 +1,2 @@
-<div uk-alert class="uk-alert-warning">
-  <FaIcon @icon="circle-info" @fixedWidth={{true}} class="margin-right-xs" />
-  {{t "ember-gwr.quarterlyClosure.components.disabledFormInfo"}}
-</div>
+<FaIcon @icon="circle-info" @fixedWidth={{true}} class="margin-right-xs" />
```
Comment (luytena) [inline addon/components/quarterly-closure/disabled-form-info.hbs:1]: Is it intentional, that the text and icon are black for linked-models?

Comment (luytena) [inline addon/components/quarterly-closure/disabled-form-info.hbs:1]: Please ensure that the text and icon are middle aligned vertically

**Comment by luytena** [comment]
Removing linked objects needs to be disabled as well.

**Comment by velrest** [comment]
image

File: addon/components/quarterly-closure/disabled-form-info.hbs (hunk 1 above)
Comment (velrest) [reply addon/components/quarterly-closure/disabled-form-info.hbs:1]: I've added a screenshot, yes it is intentional. I think it is not needed to have a flashy text there.

Comment (velrest) [reply addon/components/quarterly-closure/disabled-form-info.hbs:1]: Due to its small size i think no `flex` is needed. Would you still add it? I think that we use flex sometimes a bit too much for stuff like this.

Comment (luytena) [reply addon/components/quarterly-closure/disabled-form-info.hbs:1]: alert

It looks slightly shifted though :sweat_smile:

Comment (luytena) [reply addon/components/quarterly-closure/disabled-form-info.hbs:1]: Yes, I took a look at it and I agree :+1:

**Comment by velrest** [comment]
> @velrest Removing linked objects needs to be disabled as well.

Done :ok_hand:

File: addon/components/quarterly-closure/disabled-form-info.hbs (hunk 1 above)
Comment (velrest) [reply addon/components/quarterly-closure/disabled-form-info.hbs:1]: Is this in ng?

Comment (luytena) [reply addon/components/quarterly-closure/disabled-form-info.hbs:1]: yes

Comment (velrest) [reply addon/components/quarterly-closure/disabled-form-info.hbs:1]: :cursing_face:

Comment (velrest) [reply addon/components/quarterly-closure/disabled-form-info.hbs]: Done

Comment (luytena) [reply addon/components/quarterly-closure/disabled-form-info.hbs]: :sweat_smile:

**Comment by luytena** [comment]
@velrest Approved

**Pull request merged**